Kernel/Random.h

```diff
@@ -68,12 +68,13 @@ class FortunaPRNG {
 
         typename CipherType::CTRMode cipher(m_key, KeySize);
 
-        auto wrapped_buffer = ByteBuffer::wrap(buffer, n);
-        m_counter = cipher.key_stream(wrapped_buffer, m_counter).value();
+        Span<u8> buffer_span { buffer, n };
```

Comment:
We have the `Bytes` typedef for `Span<u8>`.

Reply:
Ah, I didn't see that.
will change.